client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -437,6 +446,14 @@ export class CheckoutThankYou extends React.Component {
 				);
 			}
 
+			if (
+				shouldShowDifmUpsell &&
+				! previousRoute.includes( `/checkout/${ selectedSiteSlug }/offer-difm/${ receiptId }` )
+			) {
+				recordTracksEvent( 'calypso_eligible_difm_upsell' );
```

Comment:
Custom Tracks event is created to be used as an exposure event for the experiment. This helps give more control on which users we want to enter into the experiment.

Reply:
Regarding my comment, you might need to split this in two.